config/_default/params.yaml

```
@@ -1,6 +1,6 @@
 homepage_featured:
   - les_meves_ajudes
-  - leximpact
+  - leximpact_socio_fiscal
```

@MattiSG It looks like we are not listing `leximpact_impot_revenu`. Is it because the application is no longer online? Or dit we forget it in this list? 
> In this PR, `leximpact` becomes `leximpact_impot_revenu` and a new showcase `leximpact_socio_fiscal` is added.
```suggestion
  - leximpact_impot_revenu
  - leximpact_socio_fiscal
```

---

As the entry says, this is not the list of every single entry but the ones that are featured on the homepage, so we will choose only one 🙂

---

I chose the one that was not marked as disused.